src/monet.d.ts

```diff
@@ -249,6 +249,7 @@ interface IEitherStatic extends IMonadStatic {
   unit: IRightStatic;
   of: IRightStatic;    // alias for unit
   pure: IRightStatic;  // alias for unit
+  attempt<E, V>(fn: () => V): Either<Error|E, V>;
```

Comment:
maybe there should be `<E extends Error, …` constraint? (also `Error|E` union would be obsolete, just `E` will do the job)

Reply:
or maybe I'm wrong, as users can throw any custom stuff… but then `Error` is redundant in union.
Maybe `attempt<E = Error, V>(fn: () => V): Either<E, V>;` default value will make it more useful!